nin/backend/generate/TemplateShaderLayer.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @constructor
+ */
+function TemplateLayer(config) {
+  this.config = config;
```

Comment:
NoiseFXlayer which i based myself on took in config. I see others take in layer.